tests/platform/broadcom/files/ser_injector.py

```diff
@@ -0,0 +1,306 @@
+#!/usr/bin/env python
+
+import subprocess
+import re
+import time
+
+# Global parameter for memory scanners
+MEMORY_SCAN_INTERVAL_USEC = int(3e5)
+MEMORY_SCAN_ENTRIES = 16384
+SRAM_SCAN_INTERVAL_USEC = int(3e5)
+SRAM_SCAN_ENTRIES = 16384
+
+DEFAULT_SER_TEST_TIME_SEC = 1200
+
+DEFAULT_SER_INJECTION_INTERVAL_SEC = 5
+DEFAULT_SYSLOG_POLLING_INTERVAL_SEC = 0.1
+
+# The following memory tables do not get corrected when a ser is injected into them! Let's take them out for now
+# until the underlying reason is answered by Broadcom
+UNTESTABLE_MEMORY_TABLES = [u'L3_ENTRY_IPV6_MULTICAST.ipipe0', u'L3_DEFIP_ALPM_IPV6_64.ipipe0', u'EGR_IP_TUNNEL_MPLS.epipe0',
+                            u'MODPORT_MAP_MIRROR.ipipe0', u'L3_DEFIP_ALPM_IPV6_128.ipipe0', u'L3_ENTRY_IPV4_MULTICAST.ipipe0',
+                            u'L3_ENTRY_IPV6_UNICAST.ipipe0', u'L3_DEFIP_ALPM_IPV4.ipipe0', u'FP_GLOBAL_MASK_TCAM.ipipe0',
+                            u'FP_STORM_CONTROL_METERS.ipipe0', u'FP_GM_FIELDS.ipipe0']
+
+# Not every memory table orrection is detected by the test case. This could be due to the
+# losses over syslog transport. Make the test iterate over different memory tables in order to cover
+# every memory table
+DEFAULT_TEST_ITERATION = 3
+
+def run_cmd(cmd):
+    '''
+    @summary: Utility that runs a command in a subprocess
+    @param cmd: Command to be run
+    @return: stdout of the command run
+    @return: stderr of the command run
+    '''
+    out = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    stdout, stderr = out.communicate()
```

Comment:
you never use stderr

Reply:
I use to debug print stderr in order to verify if error exists